sklearn/manifold/tests/test_spectral_embedding.py

```diff
@@ -181,3 +182,13 @@ def test_connectivity(seed=36):
     assert_equal(_graph_is_connected(graph), True)
     assert_equal(_graph_is_connected(csr_matrix(graph)), True)
     assert_equal(_graph_is_connected(csc_matrix(graph)), True)
+
+
+def test_spectral_embedding_deterministic(seed=36):
+    """Test that Spectral Embedding is deterministic"""
+    random_state = np.random.RandomState(seed)
```

Comment:
But other tests also used the same ```seed``` parameter and used it for the same purpose I did, Please tell me why we did that in rest ?